sphinx/source/docs/dev_guide/bokehjs.rst

```diff
@@ -121,30 +121,47 @@ Use ``node make help`` to list all available commands.
 You can use ``tsc`` directly for error checking (e.g. in an IDE). However, don't use
 it for code emit, because we rely on AST transforms to produce viable library code.
 
+.. _devguide_bokehjs_development_testing:
+
 Testing
 ~~~~~~~
 
-BokehJS testing is performed with the ``node make test`` command. You can run individual
-test suites with ``node make test:suite_name``. Known tests suites are:
+BokehJS comes with its own suites of tests and testing framework. All tests for BokehJS
+use ``describe()`` and ``it()`` functions. Unit tests are written using Chai "expect"
+style. [TBD: How are style/setup different for visual integration tests?]
+
+To launch BokehJS tests, run ``node make test`` from within the
+:bokeh-tree:`bokehjs/test` directory.
+
+Instead of running all available BokehJS tests, you can also run individual test
+suites with ``node make test:suite_name``. Available tests suites are:
+
+* ``node make test:codebase``: Codebase tests checking file size limits
+* ``node make test:defaults``: Tests checking whether the defaults in Bokeh's
+  Python models match those of Bokeh's TypeScript models
+* ``node make test:unit``: Unit tests for BokehJS
+* ``node make test:integration``:
+  :ref:`Visual integration tests <devguide_bokehjs_development_visual_testing>`
+  comparing generated plots against a set of baseline files
```

Comment:
@mattpap Is this a somewhat accurate summary of what the individual test suites are doing?